### DISCUSSION TITLE
BMS: BLE not working after 2 weeks

### DISCUSSION THREAD

**Discussion by mager33** [syssi/esphome-basen-bms#22] [General]
No way to reach my BMS... neither phone/app, nor ESPhome. 
I disconnected the battery shortly, but that did not help.
There is not way to access the BMS from outside.
Any ideas?
Best, Martin

**Comment by syssi** [comment]
This looks like a similar topic: https://www.akkudoktor.net/forum/bms-batterie-management-monitoring-system/bms-enstschlafen-laesst-sich-nicht-aufwecken/

**Reply by syssi** [reply]
@mager33 Any bad or good news?

**Reply by mager33** [reply]
No, still not working. I am reluctant to cut the battery pack open (some people did that, disconnected BT module or whole BMS).

**Reply by syssi** [reply]
Did you try to contact the manufacturer? It looks like this issue is pretty common.

**Reply by mager33** [reply]
They either send silly ideas and do not respond when asking for an engineer... still a bit reluctant to cut the case open with an osciallation saw.
